dropbox.sh

```diff
@@ -1,4 +1,6 @@
-rm -f ~/.dropbox_uploader
+#!/bin/bash
+
+rm -rfv ~/.dropbox_uploader
```

Comment:
because it does no harm and was a way to make travis rebuild the thing ;) If it annoys you I can change them all back